app/scripts/modules/kubernetes/cluster/configure/CommandBuilder.js

```diff
@@ -168,7 +168,8 @@ module.exports = angular.module('spinnaker.kubernetes.clusterCommandBuilder.serv
         return {
           fromTrigger: true,
           repository: trigger.repository,
-          registry: trigger.registry,
+          account: trigger.account,
```

Comment:
wasn't too clear where this is being used. would imagine the image selection in kubernetes would also need to be updated to have organizational filtering